src/libraries/System.Linq/src/System/Linq/LeftJoin.cs

```diff
@@ -0,0 +1,78 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+
+namespace System.Linq
+{
+    public static partial class Enumerable
+    {
+        public static IEnumerable<TResult> LeftJoin<TOuter, TInner, TKey, TResult>(this IEnumerable<TOuter> outer, IEnumerable<TInner> inner, Func<TOuter, TKey> outerKeySelector, Func<TInner, TKey> innerKeySelector, Func<TOuter, TInner?, TResult> resultSelector) =>
```

Comment:
We should add XML doc comments to all of the new public APIs.

Reply:
Ah, I was following the existing code on this (e.g. Join has no xmldocs) - are we already in a place where the API docs are generated from the xmldocs?

In any case, I'll take care of this.

Reply:
> are we already in a place where the API docs are generated from the xmldocs?

Yes and no.

Some libraries have the src as the source of truth for docs. Unfortunately, System.Linq isn't yet one of them. However, even for such libraries, we add XML comments for all new surface area, and those comments then seed the docs for those methods, even if they're not then the source of truth moving forward.

Reply:
#111177